### PR TITLE
docs(toh-4/ts): copyedits

### PR DESCRIPTION
**Pull request by chalin**
Contributing to #1628.

### CODE REVIEW

File: public/docs/_examples/toh-4/ts/app/hero.service.ts [hunk 1]
```diff
@@ -12,8 +12,8 @@ export class HeroService {
   getHeroes() {
     return Promise.resolve(HEROES);
   }
-  // #enddocregion get-heroes
-  // #enddocregion just-get-heroes
+  // #enddocregion get-heroes, just-get-heroes
+  // #enddocregion
```
Comment (Foxandxss) [inline public/docs/_examples/toh-4/ts/app/hero.service.ts]: Shouldn't this one disappear?

**Comment by wardbell** [comment]
LGTM.  I don't know about @Foxandxss question but if we messed up, we can fix it.

File: public/docs/_examples/toh-4/ts/app/hero.service.ts (hunk 1 above)
Comment (chalin) [inline public/docs/_examples/toh-4/ts/app/hero.service.ts]: I've always assumed that we could not specify the unnamed docregion when other docregions are being named. But I could be wrong (I'll test that out some time).

**Comment by chalin** [comment]
No, it would not mess anything up. I think that Jesus was commenting about a potentially unnecessary extra line (i.e., have two `#enddocregion` tags instead of one).

File: public/docs/_examples/toh-4/ts/app/hero.service.ts (hunk 1 above)
Comment (chalin) [inline public/docs/_examples/toh-4/ts/app/hero.service.ts]: Had a conversation with @Foxandxss in a separate thread, and he is ok with the change made here. FYI.

**Pull request closed without merging**